gas/testsuite/gas/riscv/zc-zcf-flw-fsw-fail-march.l

```diff
@@ -11,4 +11,3 @@
 .*: Error: unrecognized opcode `fsw fs2,0\(sp\)'
 .*: Error: unrecognized opcode `fsw fs2,100\(sp\)'
 .*: Error: unrecognized opcode `fsw fs2,248\(sp\)'
-
```

Comment:
Looks like this is now a file without an end of line? If so I would leave the end of line there, otherwise if it is two end of lines remove the latter.